Metrics/risk.md

```diff
@@ -1,5 +1,5 @@
-# Risk
+### Risk
 
-The Risk category informs how much risk an OSS community might pose. The evaluation of risk depends on situation and purpose. 
+he Risk category informs how much risk an OSS community might pose. The evaluation of risk depends on situation and purpose.
```

Comment:
Spelling error introduced at beginning of Line 3

Reply:
I went ahead and fixed it.

Reply:
thanks